bldsys/cmake/component_helpers.cmake

```diff
@@ -39,13 +51,15 @@ function(vkb__register_component)
         if(TARGET_INCLUDE_DIRS)
             target_include_directories("vkb__${TARGET_NAME}" PUBLIC ${TARGET_INCLUDE_DIRS})
         endif()
-        
+
         if(MSVC)
             target_compile_options("vkb__${TARGET_NAME}" PRIVATE /W4 /WX)
         else()
             target_compile_options("vkb__${TARGET_NAME}" PRIVATE -Wall -Wextra -Wpedantic -Werror)
         endif()
-    else() ## Create interface library
+
+        set_property(TARGET vkb__${TARGET_NAME} PROPERTY FOLDER "components/${TARGET_NAME}")
```

Comment:
Using just "components" as folder would remove the additional hierarchy in the solution.